### PR TITLE
Pass through color from intern to info call

### PR DESCRIPTION
**Pull request by gsaraf**
Propagate the requested color from the intern page to the info call.

### CODE REVIEW

File: juggler/job.go [hunk 1]
```diff
@@ -25,6 +25,7 @@ type Job struct {
 	FileContent  string             `json:"file_content"`
 	Owner        string             `json:"owner"`
 	Status       JobStatus          `json:"status"`
+	Color        string             `json:"requested_color"`
```
Comment (leoleovich) [inline juggler/job.go]: Just color please :)

**Review by leoleovich — COMMENTED** [review]
Please also update "fakejuggler" as it is used to test UI

**Review by leoleovich — CHANGES_REQUESTED** [review]
Per comments

**Comment by gsaraf** [comment]
Updated code - is that what you meant?

Thanks!

File: juggler/job.go (hunk 1 above)
Comment (gsaraf) [reply juggler/job.go]: Done in update below.

**Comment by leoleovich** [comment]
Thanks!

**Pull request merged**